builder.go

```diff
@@ -580,6 +582,15 @@ func ParseDockerignore(root string) ([]string, error) {
 	return excludes, nil
 }
 
+// ParseDockerIgnore returns a list of the excludes in the .containerfile or .dockerignore file.
+func ParseDockerignore(root string) ([]string, error) {
+	excludes, err := ParseIgnore(filepath.Join(root, ".containerignore"))
+	if err != nil && !os.IsNotExist(err) {
+		excludes, err = ParseIgnore(filepath.Join(root, ".containerignore"))
```

Comment:
I think you want this here:
```suggestion
		excludes, err = ParseIgnore(filepath.Join(root, ".dockerrignore"))
```

Reply:
Oops

Reply:
`.dockerignore`, not `.dockerrignore`

Reply:
double oops!